signature-envelope-jws.md

```diff
@@ -0,0 +1,158 @@
+# JWS Signature Envelope
+
+JWS ([RFC7515](https://datatracker.ietf.org/doc/html/rfc7515)) is a JSON based envelope format for digital signatures over any type of payload (e.g. JSON, binary). Notary v2 uses JWS as a supported signature format. It specifically uses the *JWS JSON Serialization* representation, which supports unsigned attributes. JWS Compact serialization, and JWT (also based on Compact serialization), do not support unsigned attributes.
+
+## Storage
+
+A JWS based signature will be persisted with a blob `mediaType` of `"application/jose+json"`
+
+Example
+
+```jsonc
+{
+   "artifactType": "application/vnd.cncf.notary.v2.signature",
+    "blobs": [
+        {
+            "mediaType": "application/jose+json",
+            "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0",
+            "size": 32654
+        }
+    ],
+    "subject": {
+        "mediaType": "application/vnd.oci.image.manifest.v1+json",
+        "digest": "sha256:73c803930ea3ba1e54bc25c2bdc53edd0284c62ed651fe7b00369da519a3c333",
+        "size": 16724
+    },
+    "annotations": {
+        "io.cncf.notary.x509.fingerprint.sha256": "[\"B7A69A70992AE4F9FF103EBE04A2C3BA6C777E439253CE36562E6E98375068C3\" \"932EB6F5598435D4EF23F97B0B5ACB515FAE2B8D8FAC046AB813DDC419DD5E89\"]"
+    }
+}
+```
+
+## JWS Payload
+
+The payload in the JWS envelope is set to BASE64URL encoded value of [Notary v2 Payload](./signature-specification.md#payload).
+
+Example of payload before BASE64URL encoding.
```

Comment:
This example does not flow from the previous text.  You were talking about BASE64URL and then give an example that is before encoding but it then implies an after example.

Reply:
Agreed, will address in next rev.